NativeApp/src/utilities/AxiosInstance.js

```diff
@@ -3,7 +3,8 @@ import Moment from 'moment';
 import deviceStorage from '../services/deviceStorage';
 
 
-const baseURL = process.env.DOMAIN;
+const baseURL = 'http://192.168.99.100:8081'
+//process.env.DOMAIN;
```

Comment:
Should this change be committed?

Reply:
woops, wasn't meant to commit that. good spot